lua/autorun/server/sv_restrict_propcore.lua

```diff
@@ -1,41 +1,40 @@
--- Propcore is allowed to everyone, but functions in the restrictedFunctions array will be restricted to devotee+ only
+-- Restricts PropCore functions based on rank
 
+local restrictedFunctions, rankRestrictedFunctions = require "sv_restricted_propcore_functions"
```

Comment:
this require wasn't working for me`[ERROR] addons/cfc_restrict_propcore/lua/autorun/server/sv_restrict_propcore.lua:3: Module not found!`
but this worked
```lua
local restrictedFunctions, rankRestrictedFunctions = include "sv_restricted_propcore_functions.lua"
```

Reply:
/shrug